urth/dashboard/converter/static/main.js

```diff
@@ -36,12 +36,33 @@ requirejs.config({
     }
 });
 
-
 requirejs(['urth/dashboard'], function(Dashboard) {
     Dashboard.init().then(function() {
-        // Ugly, because we're tying dashboards specifically to urth_widgets
-        // but we don't have another way to detect and determine if / how these
-        // should be setup at the moment.
+        // Ugly, because we're special-casing declarative widget support, but
+        // they need to be pre-loaded onto the page before we begin executing
+        // any notebook code that depends on them.
+
+        // Declarative widgets 0.2.x subclass the WidgetModel base class
+        // to guarantee message sequence. They register this new base with 
+        // the widget manager. Because the Thebe build includes these 
+        // modules and initializes the manager singleton, we need to map 
+        // them across from the Thebe loader to our loader to avoid making
+        // second, independent copies.
+        // All this goes away when we can use jupyter-js-services and 
+        // ipywidgets independently.
+        define('nbextensions/widgets/widgets/js/manager', function() {
+            return {
```

Comment:
It might be more "correct" to pass in the `.../widgets/js/manager` reference as a parameter and use that below instead of the `IPython` global.  Same for the 2nd `define` below.